hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieLogFile.java

```diff
@@ -83,11 +83,12 @@ public String getFileName() {
     public HoodieLogFile rollOver(FileSystem fs) throws IOException {
         String fileId = getFileId();
         String baseCommitTime = getBaseCommitTime();
+        String extension = "." + FSUtils.getFileExtensionFromLog(path);
```

Comment:
Hmmm I remember leaving a comment around not overloading the HoodieLogFile to be used for the ArchiveLog..  This is meant for representing the log with real data.. Please extend this class with a different extension or use another one.. do things like `getBaseCommitTime` make sense in the context of archive log?